bvm/ballerina-core/src/main/java/org/ballerinalang/util/metrics/Constants.java

```diff
@@ -0,0 +1,30 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package org.ballerinalang.util.metrics;
+
+/**
+ * Defines the types of the metrics supported.
+ */
+public class Constants {
```

Comment:
Shall we rename this class to MetricsConstants or something similar. We are in the process of renaming existing Constants classes too (https://github.com/ballerina-platform/ballerina-lang/issues/4558)